src/iam/chainConfig.ts

```diff
@@ -31,7 +30,6 @@ export interface MessagingOptions {
  */
 export const chainConfigs: Record<number, ChainConfig> = {
   [VOLTA_CHAIN_ID]: {
-    rpcUrl: "https://volta-rpc-vkn5r5zx4ke71f9hcu0c.energyweb.org/",
```

Comment:
Sorry, the story description wasn't very clear. All we need to do here is replace "https://volta-rpc-vkn5r5zx4ke71f9hcu0c.energyweb.org/" with "https://volta-rpc.com". We don't need to actually remove the `rpcUrl` property 😄

Reply:
ah omg:D actually I spent most of the time getting familiar with code. and waste max. 1 hour on just coding :D so no worries, will revert back the rpcUrl config and also will change the URL as specified.